app/components/Mask.js

```diff
@@ -8,6 +8,8 @@ import Flex from "components/Flex";
 type Props = {|
   header?: boolean,
   height?: number,
+  minWidth?: number,
```

Comment:
Allows tweaking the size of the placeholder mask for specific usecases where content length is known